modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -0,0 +1,353 @@
+= Mule Upgrade Tool
+
+*Latest version: 1.0.0*
+
+include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool.adoc[tag=intro]
+
+The Mule upgrade tool runs in any environment that is capable of running a standalone Mule instance. This tool can update only standalone Mule instances that are running any Mule 4 version.
+//You can upgrade local Mule instances to the latest available version or to a particular Mule distribution that you have previously downloaded and unpacked.
+You can also use the Mule upgrade tool to update Mule instances located in different folders.
+
+As part of the upgrade process, the Mule upgrade tool also updates the Runtime Manager agent version if it is installed in your current Mule instance.
+
+The Mule upgrade tool does not support upgrading clustered Mule instances. For clustered environments, upgrade each of the nodes manually by following the instructions in xref:release-notes::mule-runtime/updating-mule-4-versions.adoc#mulerunvers[Upgrading an On-Premises Mule Instance Managed Through Runtime Manager]
+
+== Before You Begin
+
+Before starting your upgrade, ensure that you have:
+
+* A currently operational Mule 4 instance, in _stopped_ status to prepare for the upgrade
+* The Mule upgrade tool inside your current Mule instance's `$MULE_HOME/tools` directory
++
+If the Mule upgrade tool is not bundled with your current Mule distribution, download the tool from the https://help.mulesoft.com/s/[Help Center^] and copy it into the `$MULE_HOME/tools` directory of your current Mule instance.
```

Comment:
Remove the line that it is not bundled with your Mule distribution. Suggested text: 

Download the Mule Runtime Upgrade tool from the https://help.mulesoft.com/s/[Help Center^] and copy it into the `$MULE_HOME/tools` directory of your current Mule instance.